### ISSUE TITLE
Cataclysm Stands/Classbar

### ISSUE THREAD

**Issue by Weslay972** [tullamods/Dominos#852]
Hello,
Cataclysm prepatch as realease and i have a problem with dominos addon on my Shaman.
I dont see the classbar totem (same on my warrior , rogue and deathknight)
Is there a solution to solve that please?

**Comment by Nekura-Neko** [comment]
I can't speak to all of this but: I self-patched my client pending an authoritative update; /dominos/bars/stanceBar.classic.lua has checks for "Addon:IsBuild('wrath')" on DK, Priest, & Warlock that you can just replace with "true" -- same for "HUNTER" which is "false" but should be "true" for Cataclysm -- this worked for my DK and Hunter.

The totemBar.lua also has an "Addon:IsBuild('wrath'))" check on it, but removing that check doesn't bring the Totem bar back; there's something more going on in there and it's making my Shaman (main) sad and confused.

I don't play rogue or warrior but their stanceBar.classic.lua setting is fixed at "true" so I'm surprised they don't work; my comparable paladin and druid work fine without my monkeypatching.

**Comment by Tuller** [comment]
The totem bar is a bit messy at the moment in Cata. It doesn't show up on initial login, but you'll see it after a `/console reloadui`

**Comment by Nekura-Neko** [comment]
> The totem bar is a bit messy at the moment in Cata. It doesn't show up on initial login, but you'll see it after a `/console reloadui`

Yeah, it was like that in Wrath too -- totally Blizzard's fault. We're used to it. Fix confirmed, and I see profiles are back too. Thank you so much! <3!

**Comment by NickNiz** [comment]
/reload isn't doing it for me. When I disable domino's the totem bar shows. But when I have domino's loaded still no totem bar after countless reloads.
